### DISCUSSION TITLE
Introspection of `pyproject.toml`?

### DISCUSSION THREAD

**Discussion by alexpdp7** [python-poetry/poetry#7521] [Q&A] [answered]
Assume I wanted to build a command that lists all scripts in my Poetry project. I shouldn't inspect `pyproject.toml`- that doesn't seem to be included in wheel dists by default. I see `entry_points.txt` is in the wheel.

Is there any "official" way to do this kind of thing?

**Comment by clintonroy** [comment, marked as answer]
https://docs.python.org/3/library/importlib.metadata.html#entry-points

With an example of looking console scripts.

**Reply by alexpdp7** [reply]
Sweet. I had to use the backport, and I had to do some additional filtering (that seems to list *all* entry points in the current Python installation), but I could do what I needed, thanks!
